rust/chains/hyperlane-cosmos/src/libs/binary.rs

```diff
@@ -0,0 +1,28 @@
+use hyperlane_core::{H160, H256, H512};
+use std::cmp;
+
+/// Convert H256 to H512
+pub fn h256_to_h512(v: H256) -> H512 {
```

Comment:
pretty sure we have utils or Into/From impls for all these already in some way and can replace these